shuffle/move/examples/main/sources/defi/Token.move

```diff
@@ -0,0 +1,49 @@
+module 0x1::Token {
```

Comment:
Can we make the module address `Sender` (`module 0x1::Token`) so that it can be published by a regular key/address pair, and not the TC account `0xB1E55ED`?

Or are we planning to include this as a module in a custom genesis? 

I'm leaning towards the `Sender` option because of ease and simplicity.